docs/data/material/guides/pickers-migration/pickers-migration.md

```diff
@@ -2,6 +2,12 @@
 
 <p class="description">@material-ui/pickers was moved to the @mui/lab.</p>
```

Comment:
I wonder if it would make more sense to update the page to remove all mentions of `@mui/lab`. Since there are no breaking changes with the migration of the lab to MUI X. I guess we could encourage the developers to directly start there, so to update the imports on this page to point `@mui/x-date-pickers`.